serverless-local.yml

```diff
@@ -0,0 +1,391 @@
+# Welcome to serverless. Read the docs
+# https://serverless.com/framework/docs/
+
+# Serverless.yml is the configuration the CLI
+# uses to deploy your code to your provider of choice
+
+# The `service` block is the name of the service
+service: hathor-explorer-service
+useDotenv: true
+# The `provider` block defines where your service will be deployed
+provider:
+  name: aws
+  runtime: python3.8
+  region: ${env:AWS_DEFAULT_REGION}
+  lambdaHashingVersion: 20201221
+  environment:
```

Comment:
Maybe duplicating the serverless.yml file is not a good thing. They could become different if someone is not careful when changing them.

What problem did you have with the `provider.vpc` config that you had to remove?